packages/trader/src/Modules/Trading/Components/Form/TradeParams/barrier.jsx

```diff
@@ -114,17 +183,17 @@ const Barrier = ({
                     classNameInput={classNames(
                         'barrier__fields-input',
                         'barrier__fields-barriers-input',
-                        `barrier__fields-barriers-${input_class}-input`,
-                        {
-                            'barrier__fields-input--is-offset': !is_absolute_barrier,
-                        }
+                        `barrier__fields-barriers-${input_class}-input`
                     )}
                     current_focus={current_focus}
+                    display_as='div'
```

Comment:
@kate-deriv The point of LabeledQuantityInputMobile is using it as an input. If we don't want to use an input, let's not modify LabeledQuantityInputMobile but instead use a div or another component that doesn't include input.